source/whats-new.txt

```diff
@@ -12,13 +12,34 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.6 <version-4.6>`
 * :ref:`Version 4.5 <version-4.5>`
 * :ref:`Version 4.4 <version-4.4>`
 * :ref:`Version 4.3 <version-4.3>`
 * :ref:`Version 4.2 <version-4.2>`
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+
+.. _version-4.6:
+
+What's New in 4.6
+-----------------
+
+New features of the 4.6 Java driver release include:
+
+- The buffer pool is now shared across all ``MongoClient`` instances. By
+  default, the buffer pool prunes buffers that have been idle for one
+  minute.
+- Added an API to specify AWS credentials for Client-Side Field Level
+  Encryption (CSFLE) the same way as specifying authentication credentials
+  using :ref:`MONGODB-AWS <mongodb-aws-auth-mechanism>`.
+- Added CSFLE library support for Apple M1 ARM-based hardware.
```

Comment:
Instead of mysteriously referring to the library as "CSFLE library", we may just identify it as `org.mongodb:mongodb-crypt` because users specify it explicitly as a dependency, i.e., are aware of its formal name.

Reply:
I agree that including the library name is useful in this instance, but the feature name is also important to keep.

Given this is page is targeted towards both Jasmin and Lucas, I think having a high level description of what the library is for is important to us as well as it otherwise becomes "mysterious" to audiences not familiar with the dependency name.